### ISSUE TITLE
Potentially Add text to Icons

### ISSUE THREAD

**Issue by u8sand** [MaayanLab/playbook-partnership#127]
**Use case 11: Related Proteins/Metabolites across DCCs**
> icons are mostly self-explanatory, but a sentences re this (e.g., which icons to click for what (save, share URL, export, etc) will help). - Mano
> Is there anyway to include 1-3 words in the icons in small font or just not enough space. - Mano

**Comment by u8sand** [comment]
@mano-at-sdsc thank you for the suggestion, I wonder if you noticed the tooltips when hovering your mouse over the icons? Do you think the text there is enough? Maybe we can add an (i) indicator to icons so people know you can hover over them.

**Comment by mano-at-sdsc** [comment]
Yes, @u8sand I see it. Yes, some indication ma be useful. If feasible, including text in small font in the icons themselves itself will be useful; if not then leave it as it is.
